awx_collection/README.md

```diff
@@ -71,6 +71,23 @@ pip install -e .
 PYTHONPATH=awx_collection:$PYTHONPATH py.test awx_collection/test/awx/
 ```
 
+### Integration
+
+The old integration tests are kept passing, but not being expanded on.
```

Comment:
Nitpick: That sentence should read "The old integration tests kept passing, ..." (remove "are")